package.json

```diff
@@ -52,6 +52,7 @@
     ]
   },
   "dependencies": {
+    "deep-freeze": "^0.0.1",
```

Comment:
There are a host of unresolved [issues](https://github.com/substack/deep-freeze/issues) with no responses on this module.  Given it is a single [12 LOC](https://github.com/substack/deep-freeze/blob/master/index.js) function, I'm wondering if we ought to just copy source into `fixd` so we can make patch updates.